grails-app/views/order/_orderItemFormDialog.gsp

```diff
@@ -362,6 +361,12 @@
     var budgetCode = $("#dlgBudgetCode").val();
     var quantity = parseInt($("#dlgQuantity").val());
     var quantityInShipments = parseInt($("#dlgQuantityInShipments").val())
+    var quantityPerUom = $("#dlgQuantityPerUom").val();
```

Comment:
```suggestion
    const quantityPerUom = $("#dlgQuantityPerUom").val();
```
was there a scope-reason you used `var`?

Reply:
there was no scope reason, I was just following the previous examples of getting the value from a  selector.
I could change it to const but it doesn't;t really make any difference in the context of this function